docs/publish-module/send-to-appstore.md

```diff
@@ -6,8 +6,55 @@ sidebar_position: 1
 ---
 import Screenshot from '@site/src/components/Screenshot';
 
+# Send Apps to App Store and TestFlight
+
 Appcircle supports sending IPA binaries to the App Store and TestFlight through the Publish module.
 
+### Prerequisites
+
+Before uploading a binary to the store, please make sure that an application listing is created in App Store Connect. Otherwise, the store upload process will fail.
+
+You also need to provide either:
+
+- An App Store Connect API Key as a P8 file along with the key ID and the issuer ID. This is the recommended authentication method. Please refer to the following document for more information about the [App Store Connect API key](https://developer.apple.com/documentation/appstoreconnectapi/creating_api_keys_for_app_store_connect_api).
+
+or
+
+- An Apple Developer ID with an app-specific password. Please refer to the following document for more information about the [App-Specific Passwords](https://support.apple.com/en-us/HT204397).
+
+### Adding an App Store Connect API Key (Recommended Method)
+
+To send apps from Appcircle, you need to provide an App Store Connect API key. To add a key, go to the [My Organization](https://docs.appcircle.io/account/my-organization) Integrations tab and press the "Add New" button (or the "Manage" button first if you have saved keys) next to the "App Store Connect API Keys" item under the Connections section. The `Add Key` screen will be displayed.
+
+[Create an API key in App Store Connect](https://developer.apple.com/documentation/appstoreconnectapi/creating_api_keys_for_app_store_connect_api) and upload it here for API authentication. Please keep this file as it is the only copy and it will be required during every store submission for security purposes.
+
+With the next option, enter the key ID and the issuer ID that can be obtained from the keys section in the [App Store Connect](https://appstoreconnect.apple.com/access/api).
+
+Then enter a user-friendly name to identify the key in the lists and press save. You can use this key in multiple apps without the need for key uploads at every upload process.
+
+<Screenshot url='https://cdn.appcircle.io/docs/assets/image (93).png' />
```

Comment:
We should update this SS since it's background from an older version of the app.

Also a descriptive file name will be better instead of "image (93).png".

---
**Note:** I guess we moved the SS from other sections of the docs to here. Only "publish module" update  will be enough since "store submit" will be removed from docs in future.